packages/dashboard/src/components/MultiCountChart/examples/MultiCountChart.Example.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { AnnotationType, MultiCountChart } from '../index';
+
+export class MultiCountChartExample extends React.Component<{}, {}> {
+  public render(): JSX.Element {
+    const rows = [
+      {
+        data: 13000,
+        bodyText: 'Flagged users',
+        annotaionText: 'Annotation Text',
+        color: 'blue',
+        type: AnnotationType.positive
+      },
+      {
+        data: 8000000,
+        bodyText: 'Risky sign-ins',
+        annotaionText: 'Decrease in the safety',
+        color: 'green',
+        type: AnnotationType.negative
+      },
+      {
+        data: 331100000,
+        bodyText: 'Risky sign-ins',
+        annotaionText: 'Annotation',
+        color: 'red',
```

Comment:
Can we show an example that references the Fabric color pallet? `color: DefaultPalette.accent` for example?